<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addannouncement</class>
 <widget class="QWidget" name="addannouncement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_airport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>376</width>
     <height>242</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>UserID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="status">
      <property name="minimumSize">
       <size>
        <width>116</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>243</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSplitter" name="sta">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QRadioButton" name="all">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>All</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="user">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>User</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>262</y>
     <width>388</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
